src/targets/cpu/lowering.cpp

```diff
@@ -501,7 +510,7 @@ struct cpu_pooling : auto_register_op<cpu_pooling<Op>>
                     }
                 });
 
-                output[i] = type(Op::final(acc, pool_size));
+                output[i] = (pool_size == 0) ? type(acc) : type(Op::final(acc, pool_size));
```

Comment:
I dont see why this is needed.

Reply:
In ceil_mode = true, case, some output elements corresponding to 0 input element. This is to avoid divided by 0 in averagePool